github-manual-merge.user.js

```diff
@@ -42,6 +42,7 @@
 			username = document.evaluate(".//*[contains(@class, 'author')]", discussionHeaderElement, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.textContent.trim();
 			var branches = document.evaluate(".//*[contains(@class, 'commit-ref')]", discussionHeaderElement, null, XPathResult.ANY_TYPE, null);
 			targetbranch = branches.iterateNext().children[1].textContent.trim();
+			branches = document.evaluate(".//*[contains(@class, 'commit-ref')]", discussionHeaderElement, null, XPathResult.ANY_TYPE, null);
```

Comment:
Oops, forgot to say in my comment on #8. This doesn't work (it only doesn't barf).